src/Converter.cpp

```diff
@@ -200,11 +261,14 @@ int convertMetaData(
     return exitStatus;
 }
 
-Converter::Converter(std::string const& basename, Typelib::Type const& type, msgpack_packer& pk, int size, int containerLimit, int verbose)
-    : data(NULL), type(type), offset(0), part(false), pk(pk), size(size),
+Converter::Converter(std::string const& basename, Typelib::Type const& type, msgpack_packer& pk, int containerLimit, int verbose)
+    : type(type), pk(pk), 
       containerLimit(containerLimit), verbose(verbose), depth(0),
       indentation(1)
 {
+    
+    debug = (verbose > 3);    
```

Comment:
Why does this need brackets?

Reply:
For clarity.

Reply:
In my opinion it should be quite clear that this is a boolean expression with a boolean return value either with or without brackets. Only complicated expressions with multiple operators and complicated execution order would justify adding brackets for clarity. It's a minor issue though and I can live with that.